apps/daffio/src/app/design/device/laptop/laptop.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
```

Comment:
Why is this a part of this PR still?

Reply:
Not sure, I thought we deleted it in the revert. I've removed it here bd0a1db